integration_tests/utils.go

```diff
@@ -124,7 +123,7 @@ func WaitForLBReady(ns, statusPath string, e *TestEnv, t *testing.T) string {
 			lbUrl, _ = RunCommandSilent(fmt.Sprintf("%s -n %s get spinsvc %s -o=jsonpath='%s'", e.KubectlPrefix(), ns, SpinServiceName, statusPath), t)
 		}
 		if lbUrl != "" {
-			_, err := RunCommandSilent(fmt.Sprintf("curl %s", lbUrl), t)
+			_, err := RunCommandSilent(fmt.Sprintf("docker run --rm --network host curlimages/curl %s", lbUrl), t)
```

Comment:
Since the integration tests are interacting with the spinnaker URL and the URL is generated within the docker network.
We need to emulate a curl call inside of the docker network, so we are using a docker image instead of the CURL command itself for it.

Reply:
This would be great as a comment in the codebase! Very useful context.

Reply:
I just added